### PR TITLE
Fix scrollview nav dots

### PR DESCRIPTION
**Pull request by PaskalevStoyan**

### CODE REVIEW

File: packages/default/scss/scrollview/_layout.scss [hunk 1]
```diff
@@ -47,29 +47,6 @@
         }
     }
 
-    kendo-scrollview.k-scrollview-wrap.k-scrollview-multidot .k-scrollview-pageable > .k-button,
-    kendo-scrollview.k-scrollview.k-scrollview-multidot .k-scrollview-pageable > .k-button,
-    .k-widget.k-scrollview.k-scrollview-multidot .k-scrollview-nav > .k-link {
```
Comment (silviyaboteva) [inline packages/default/scss/scrollview/_layout.scss:52]: Check if removing these styles will break the jQuery widget.

File: tests/visual/src/misc/colorpicker-rtl.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
                     </div>
                 </div>
             </section>
-            <section class="k-rtl">
```
Comment (silviyaboteva) [inline tests/visual/src/misc/colorpicker-rtl.html:63]: Is this the fix for the ColorPicker that we sync before? If yes, then please cherry-pick it to the `fix-color-gradient-rtl` branch.

File: packages/default/scss/colorpicker/_layout.scss [hunk 3]
```diff
@@ -392,12 +392,11 @@
             margin-top: ($colorpicker-margin * 2);
             text-align: center;
         }
-    }
 
-    .k-rtl,
```
Comment (silviyaboteva) [inline packages/default/scss/colorpicker/_layout.scss:397]: This commit also should be in `fix-color-gradient-rtl` branch.

File: packages/default/scss/scrollview/_layout.scss (hunk 1 above)
Comment (PaskalevStoyan) [reply packages/default/scss/scrollview/_layout.scss:52]: I couldn't find the same class pattern on the jQuery website when inspecting their scrollView demos.

**Review by silviyaboteva — APPROVED** [review]
Great work on this! 🚀

**Review by joneff — APPROVED** [review]

**Pull request merged**